src/app/profile/settings/feature-opt-in/feature-opt-in.component.html

```diff
@@ -0,0 +1,95 @@
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="list-group-item">
+          <div class="list-view-pf-checkbox">
+            <input type="radio" name="Beta Features" value="beta" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+          </div>
+          <div class="list-view-pf-main-info">
+            <div class="list-view-pf-left">
+              <span class="fa list-view-pf-icon-sm" style="font-family: 'Open Sans'; font-size:18px; line-height: 1.3"> ß </span>
+            </div>
+            <div class="list-view-pf-body">
+              <div class="list-view-pf-description">
+                <div class="list-group-item-heading">
+                  Beta Features
+                </div>
+                <div class="list-group-item-text">
+                  Experience various features that are ready for beta testing.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="list-group-item">
+          <div class="list-view-pf-checkbox">
+            <input type="radio" name="Experimental Features" value="experimental" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+          </div>
+          <div class="list-view-pf-main-info">
+            <div class="list-view-pf-left">
+              <span class="fa fa-flask list-view-pf-icon-sm"></span>
+            </div>
+            <div class="list-view-pf-body">
+              <div class="list-view-pf-description">
+                <div class="list-group-item-heading">
+                  Experimental Features
+                </div>
+                <div class="list-group-item-text">
+                  This is the default and current release version of the product.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="list-group-item">
+          <div class="list-view-pf-checkbox">
+            <input type="radio" name="Internal Features" value="internal" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+          </div>
+          <div class="list-view-pf-main-info">
+            <div class="list-view-pf-left">
+              <span class="fa fa-lock list-view-pf-icon-sm"></span>
+            </div>
+            <div class="list-view-pf-body">
+              <div class="list-view-pf-description">
+                <div class="list-group-item-heading">
+                  Internal Experimental Features
+                </div>
+                <div class="list-group-item-text">
+                  These experimetnal features are released to internal employees only.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="col-xs-2 col-sm-6 col-md-6"></div>
```

Comment:
What is this empty `<div>` for?

Reply:
I thought you had to introduce empty divs so that all the columns added up to twelve, I see that it's not necessary !